### ISSUE TITLE
Templating "Seeting pods page"

### ISSUE THREAD

**Issue by jonakangou** [pods-framework/pods#4100]
hi,

This is a request. I would like to templating page made when you create Setting page with pods.

Mean i would like to make same i do with "template add-on" but for setting page. So put html/css on a template and said pods use this template when you show seeting page made with pods

Exemple on file join that i would like to do
**Original setting page** 
![original-setting-page](https://cloud.githubusercontent.com/assets/25192271/24579453/41659bd6-16f6-11e7-9d31-f1250dc69517.JPG)

**What i want to do**
![what-i-want-setting-page](https://cloud.githubusercontent.com/assets/25192271/24579457/4b9d8f82-16f6-11e7-9a55-35d5a88ce284.jpg)

it's for have a more sexy user admin page when use pods in setting. Don't want to have a form engine just have an Editor that i can put HTM/CSS and then said "_this is the template to use for show this setting in admin_". Like you do actually for "normal Pods"

Thanks for your interress of it even it's not for me and i thing for you a priority to have this request on PODs

**Comment by JoryHogeveen** [comment]
@sc0ttkclark I think this could be done when the field groups are ready right?
Milestone 2.8??

**Comment by jimtrue** [comment]
Yep, if we do Field Groups with 2.8, then this would be possible with fields in Pods Settings Pages.

Doing a 'tab interface' though, might be a little different. Option for display?

**Comment by JoryHogeveen** [comment]
Yeah I'd say this could be an option for setting pages (Admin UI Style) since they don't use metaboxes.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark Thoughts?

**Comment by sc0ttkclark** [comment]
I'll see if we can manage to make this happen with Pods 2.9.

**Comment by sc0ttkclark** [comment]
So far field groups are using headings right now but in the future we now have the ability to have each field group as their own tabs. Since we have the heading field type, people can still add headings however they want and the groups can use the tabbed interface.
